dipy/denoise/nlmeans_block.pyx

```diff
@@ -277,25 +278,25 @@ cdef double _local_mean(double[:, :, :]ima, int x, int y, int z) nogil:
                                           pz - 1 if pz >= dims2 else pz))
                 ss += ima[px, py, pz]
 
-    return ss / 27.0
+    return ss / vol
 
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
 @cython.cdivision(True)
-cdef double _local_variance(double[:, :, :] ima, double mean, int x, int y, int z) nogil:
+cdef double _local_variance(double[:, :, :] ima, double mean, int x, int y, int z, int p) nogil:
```

Comment:
This is going over 80 characters. Please break into two lines, by adding a line-break somewhere.